build/openwisp_nginx/global.conf

```diff
@@ -0,0 +1,4 @@
+# Global settings across valid across all
+# configration files
+ 
+server_names_hash_bucket_size $NGINX_SERVER_NAME_HASH_BUCKET;
```

Comment:
At first read this gives me the impression that this file could contain NGINX global configs.  However, everything in conf.d get's nested at the top of the `http {}` context.  Not sure it matters but the name global may be confusing.

Reply:
I agree. Changed file references to `http.conf`. :-)